obal/data/roles/diff_package/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- block:
```

Comment:
Given this `block` now only contains one task, should we remove the `block` completely?